src/app/clusters/access-control-server/access-control-server.cpp

```diff
@@ -645,6 +645,13 @@ bool emberAfAccessControlClusterReviewFabricRestrictionsCallback(
             entry.restrictions.push_back(restriction);
         }
 
+        if (restrictionIter.GetStatus() != CHIP_NO_ERROR)
+        {
+            ChipLogError(DataManagement, "AccessControlCluster: invalid ARL data");
+            commandObj->AddStatus(commandPath, Protocols::InteractionModel::Status::InvalidValue);
```

Comment:
```suggestion
            commandObj->AddStatus(commandPath, Protocols::InteractionModel::Status::InvalidCommand);
```

since I think this falls under:

> incoming data cannot be mapped to the expected data type for a field

in the spec.  The other GetStatus() check should be doing the same; I had missed that, sorry.

Reply:
Ahh, thanks for showing me what I missed in the spec.  Fixed.